workbench/app/Console/StatCommand.php

```
@@ -247,6 +195,98 @@
+            $res = [
+                'code' => $row->code,
+                'name' => $row->name,
+                'latitude' => $row->latitude ? (float) $row->latitude : null,
+                'longitude' => $row->longitude ? (float) $row->longitude : null,
+            ];
+
+            if ($table === 'villages') {
+                $res['postal_code'] = $row->postal_code;
+            }
+
+            foreach ($res as $field => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (Unused Code Rules)

(UnusedLocalVariable)